internal/state/store.go

```diff
@@ -59,13 +60,20 @@ func abciResponsesKey(height int64) []byte {
 
 // stateKey should never change after being set in init()
 var stateKey []byte
+var tmpABCIKey []byte
 
 func init() {
 	var err error
 	stateKey, err = orderedcode.Append(nil, prefixState)
 	if err != nil {
 		panic(err)
 	}
+	// temporary extra key before consensus param protos are regenerated
+	// TODO(wbanfield) remove in next PR
+	tmpABCIKey, err = orderedcode.Append(nil, int64(10000))
```

Comment:
This is a temporary workaround. Once the proto is regenerated, this will be completely unnecessary.